documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus
+  name: infinispan-monitoring # Names the service monitor.
+  namespace: {example_crd_name} # Namespace to install the service monitor.
```

Comment:
```suggestion
  name: {example_crd_name} # Names the service monitor.
  namespace: infinispan-monitoring # Namespace to install the service monitor.
```

Reply:
I'm not convinced this needs to be in the "infinispan-monitoring" namespace. When I set up monitoring, I put the `ServiceMonitor` in the same namespace as my cluster.

Reply:
`namespace` and `name` are mixed up